src/tests/ie_test_utils/common_test_utils/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ function(add_common_utils ADD_TARGET_NAME)
                 PUBLIC
                     gtest
                     gtest_main
+                    gmock
```

Comment:
Why did you add a new dependency?

Reply:
gmock `EXPECT_THAT` macros is used in https://github.com/openvinotoolkit/openvino/blob/974650ba6d7a0a17cddcfd944c285e47663f919b/src/tests/ie_test_utils/common_test_utils/test_assertions.hpp#L82